src/uct/tcp/tcp_net.c

```diff
@@ -5,12 +5,12 @@
  * See file LICENSE for terms.
  */
 
-#include "tcp.h"
-
 #ifdef HAVE_CONFIG_H
 #  include "config.h"
 #endif
 
+#include "tcp.h"
```

Comment:
unrelated?

Reply:
> unrelated?

related, but this is workaround for issue. will apply another fix, more accurate